src/client/sandbox/native-methods.ts

```diff
@@ -1,4 +1,6 @@
 /*global Document, Window */
+const NATIVE_CODE_RE = /\[native code]/;
```

Comment:
Let's create a static method in the NativeMethods sandbox and use it everywhere:
```js
NativeMethods.isNativeCode(code)
```